docs/guides/preinstall.md

```diff
@@ -0,0 +1,81 @@
+
+## Partition with Disk Utility
+
+In macOS Disk Utility, [create a partition](https://support.apple.com/guide/disk-utility/dskutl14027/mac). The file system format doesn't matter, but pick the amount of space that you want for Linux. You won't be able to resize your APFS partitions from a Linux installer, so you must make space now. **When prompted to, do not create an APFS volume for Linux**, you want a partition.
+
+!!! note "Triple booting with Linux and Windows"
+    Refer to [this guide](https://wiki.t2linux.org/guides/windows/) to make sure you will be able to boot both Windows and Linux.
+
+## Create Linux installation USB
+
+### Selecting an ISO
+
+If there is an ISO with T2 support for your distro, you can download it here:
+
+| Linux Distribution | Install ISO with T2 support |
+| ------------------ | --------------------------- |
+| Arch Linux         | https://github.com/t2linux/archiso-t2/releases/latest |
+| EndeavourOS        | https://github.com/t2linux/EndeavourOS-ISO-t2/releases/latest |
+| Fedora Linux       | https://github.com/mikeeq/mbp-fedora |
+| Gentoo             | Please refer to this [page](https://wiki.t2linux.org/distributions/gentoo/installation/) |
+| Manjaro            | https://github.com/NoaHimesaka1873/manjaroiso-t2/releases/latest |
+| NixOS              | https://github.com/kekrby/nixos-t2-iso/releases/latest TODO: Instructions for combining iso parts |
+| Ubuntu             | https://github.com/AdityaGarg8/T2-Ubuntu/releases/latest |
+
+For other distros, you can download the distro's normal install ISO, but you will have to use an external USB keyboard and mouse for this install process, and you may need a wired internet connection.
+
+### Copying the ISO to the USB
+
+You can use [Belana Etcher](https://www.balena.io/etcher/) to copy the Linux install image to your USB.
+
+!!! hint "`dd`"
+    If you are familiar with the `dd` tool and the command line, you can use that instead. Use `diskutil list` to find the correct disk, then `sudo dd if=path/to/filename.iso of=/dev/rdiskX bs=1m`. Press `control-T` while it is running to make it show its current progress. **Make sure you select the correct disks.**
+
+While the installation image is being written to the USB, you can copy Wi-Fi firmware, but don't follow the steps after that until it has finished writing the image.
+
+## Copy Wi-Fi firmware
+
+Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access these files and then install them.
+
+Follow the instructions in this section for on macOS [here](https://wiki.t2linux.org/guides/wifi/#on-macos). Don't boot into Linux and follow the subsequent instructions yet.
```

Comment:
Don't boot into Linux ...

This line is a bit confusing. What exactly we wish to convey here?

Reply:
at this point we just want them to put the firmware on the efi partition, so the instructions in that section of the wifi guide

4. When the script shall run successfully, it shall ask you to follow either of the two options mentioned in the On Linux section, on Linux.
5. Boot into Linux.

Shouldn't be followed yet, thus don't boot linux yet

Reply:
Well obviously we cannot boot into Linux if it’s not installed. Maybe we can write that the Linux bit can be followed after Linux has installed successfully.

Reply:
d03aadef42316831995bd1b0dfc562ca233ae497

> The sections of that page for on Linux can be followed later, once you've successfully installed Linux.

Reply:
Instead of `for on`, could we do `to be followed on`?